src/components/list/List.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useSelector } from '../../hooks/markDownContext';
+import { getMarkDownTitles } from '../../selectors/markDownSelectors';
+import { Link } from 'react-router-dom';
+
+const List = () => {
+  const markDownTitles = useSelector(getMarkDownTitles);
+
+  const markDownElements = markDownTitles.map((title, i) => (
+    <li key={i}>
+      <Link to={`/${i}`}> <p>{title}</p></Link>
+      <button>delete!!!</button>
```

Comment:
you dont have delete functionality yet. wait to add this until you have the delete functionality written